frontend/src/main/java/macrobase/runtime/resources/AnalyzeResource.java

```diff
@@ -59,7 +62,17 @@ public AnalysisResponse getAnalysis(AnalysisRequest request) {
                 conf.set(MacroBaseConf.DATA_LOADER_TYPE, MacroBaseConf.DataIngesterType.CSV_LOADER);
             }
 
-            List<AnalysisResult> results = new BasicBatchedPipeline().initialize(conf).run();
+
+            Class c = Class.forName(conf.getString(MacroBaseConf.PIPELINE_NAME, MacroBaseDefaults.DEFAULT_PIPELINE));
```

Comment:
Can we create a private final static member `defaultPipeline` inside `AnalyzeResource` and use this instead of adding it to `MacroBaseDefaults`?